### PR TITLE
build: upgrade gjson to v1.6.7

### LINKED ISSUE #1353: There is a vulnerability in gjson v1.6.1,upgrade recommended
https://github.com/apache/apisix-dashboard/blob/e04c6516ec1101a758f076f2df87ec4022adcb29/api/go.mod#L38

CVE-2020-35380  CVE-2020-36066  CVE-2020-36067

Recommended upgrade version：
v1.6.7

### PR DESCRIPTION
**Pull request by ThanoshanMV**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [x] Backport patches

- Related issues
fixes #1353 

- How to fix?
Upgraded version gjson to `v1.6.7`

### CODE REVIEW

File: api/go.sum [hunk 1]
```diff
@@ -377,8 +377,12 @@ github.com/stretchr/testify v1.6.1 h1:hDPOHmpOpP40lSULcqw7IrRb/u7w6RpDC9399XyoNd
 github.com/stretchr/testify v1.6.1/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/h/Wwjteg=
 github.com/tidwall/gjson v1.6.1 h1:LRbvNuNuvAiISWg6gxLEFuCe72UKy5hDqhxW/8183ws=
 github.com/tidwall/gjson v1.6.1/go.mod h1:BaHyNc5bjzYkPqgLq7mdVzeiRtULKULXLgZFKsxEHI0=
```
Comment (nic-chen) [inline api/go.sum:379]: we could run `go mod tidy` to remove the old version in `go.sum`, thanks.

Comment (ThanoshanMV) [reply api/go.sum:379]: Okay, I'll do it.

**Comment by juzhiyuan** [comment]
cc @starsz to review

File: api/go.mod [hunk 2]
```diff
@@ -21,6 +21,7 @@ require (
 	github.com/gin-contrib/sessions v0.0.3
 	github.com/gin-contrib/static v0.0.0-20200916080430-d45d9a37d28e
 	github.com/gin-gonic/gin v1.6.3
+	github.com/go-playground/assert/v2 v2.0.1
```
Comment (membphis) [inline api/go.mod:24]: why do we need to add this library? 

I think that we can only update `github.com/tidwall/gjson` is enough 

@nic-chen please confirm

**Comment by membphis** [comment]
and the front e2e failed: https://github.com/apache/apisix-dashboard/pull/1366/checks?check_run_id=1756969309#step:8:289

need @juzhiyuan confirm

File: api/go.mod (hunk 2 above)
Comment (nic-chen) [reply api/go.mod:24]: @ThanoshanMV  we don't need this lib, please remove it. thanks.

Comment (ThanoshanMV) [reply api/go.mod:24]: I'll remove it.

**Review by nic-chen — APPROVED** [review]

**Comment by membphis** [comment]
https://github.com/apache/apisix-dashboard/pull/1364

has been merged, @ThanoshanMV you can merge the master branch, fixed the e2e test cases of FE

**Review by starsz — APPROVED** [review]

**Comment by juzhiyuan** [comment]
@ThanoshanMV Please sync with the latest codes from the master branch.

**Comment by ThanoshanMV** [comment]
Okay, I'll do it @juzhiyuan

**Comment by membphis** [comment]
> Okay, I'll do it @juzhiyuan

I helped you do this right now ^_^

**Review by membphis — APPROVED** [review]

**Pull request merged**